llvm/lib/BitstreamCAS/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+# The writer is header-only.
```

Comment:
You don't need this sub-directory if it is header only. This can be added when `.cpp` is needed.